package.json

```diff
@@ -109,6 +109,7 @@
     "build": "node ./scripts/rollup/build-all-release-channels.js",
     "build-combined": "echo 'build-combined is deprecated. yarn build instead.'",
     "build-for-devtools": "cross-env RELEASE_CHANNEL=experimental yarn build react/index,react/jsx,react-dom/index,react-dom/unstable_testing,react-dom/test-utils,react-is,react-debug-tools,scheduler,react-test-renderer,react-refresh,react-art --type=NODE && cp -r ./build/node_modules build/oss-experimental/",
+    "build-for-devtools-windows": "cross-env RELEASE_CHANNEL=experimental yarn build react/index,react/jsx,react-dom/index,react-dom/unstable_testing,react-dom/test-utils,react-is,react-debug-tools,scheduler,react-test-renderer,react-refresh,react-art --type=NODE -and xcopy /s /i build\\node_modules build\\oss-experimental/",
```

Comment:
It looks to me that the only thing that should be cross-platform here is the `cp` command. I think we should not have a separate command for windows, but:
- Implement a cross-platform `cp` (maybe there is a package for it or we already doing it somewhere in this repo)
- Use this implementation in scripts, instead of `exec` calls

Reply:
Hi I had found a way but did not want to add dependencies , here is the npm package i found https://www.npmjs.com/package/shx, if you allow me I can I add this to the react extensions package.json or maybe the react package.json

Reply:
Also the cp command in build-for-devtools script seems unnecessary because the build folder is moved to a temporary directory.

Reply:
> Also the cp command in build-for-devtools script seems unnecessary because the build folder is moved to a temporary directory.

If so, then this can be safely removed

Reply:
Alright I will remove it. is the workaround for windows in exec commands okay?

Reply:
> Hi I had found a way but did not want to add dependencies , here is the npm package i found https://www.npmjs.com/package/shx, if you allow me I can I add this to the react extensions package.json or maybe the react package.json

I think we can use https://www.npmjs.com/package/shelljs. Since this will be used in root-level scripts, this dependency should be defined in root's `package.json`

Reply:
ok sounds good, I will add that and move these commands to shell js. I was wondering if other exec commands not related to this issue might need to be changed?

Reply:
> ok sounds good, I will add that and move these commands to shell js. I was wondering if other exec commands not related to this issue might need to be changed?

Lets change only the ones that are required to have https://github.com/facebook/react/issues/27193 fixed, otherwise it would significantly increase the scope of the changes

Thank you!